tests/CMakeLists.txt

```diff
@@ -198,6 +198,18 @@ set_tests_properties(test_log_negative
 add_test(NAME test_ip_secondary
          COMMAND ${PROJECT_SOURCE_DIR}/tests/ipsecondary_test.py --build-dir ${PROJECT_BINARY_DIR})
 
+add_executable(aktualizr-cycle-simple aktualizr_cycle_simple.cc)
+target_link_libraries(aktualizr-cycle-simple aktualizr_static_lib ${AKTUALIZR_EXTERNAL_LIBS})
+aktualizr_source_file_checks(aktualizr_cycle_simple.cc)
+
+if(FAULT_INJECTION)
+    # run with a very small amount of tests on CI, should be more useful when
+    # run for several hours
+    add_test(NAME test_io_failure COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/test_io_failure.py
+        --akt-repo $<TARGET_FILE:aktualizr-repo> --akt-test $<TARGET_FILE:aktualizr-cycle-simple> -n 50 -j 10
```

Comment:
How long will 50 runs take? I tried to look in CI but I can't find it.

Reply:
CI failed sorry, forgot something. I've just relaunched it

Reply:
Took 10s on gitlab, 15s on jenkins. Let's see on Travis :)